doc/cephfs/quota.rst

```diff
@@ -48,7 +48,7 @@ To view quota limit::
    system call. Instead, a specific CephFS extended attribute can be viewed by
    running ``getfattr /some/dir -n ceph.<some-xattr>``.
 
-To remove a quota, set the value of extended attribute to ``0``::
+To remove a quota, remove the respective extended attribute or set the value to ``0``::
```

Comment:
i know it's quite trivial but how about adding example illustrating removal using `setfattr -x`, maybe helpful for naive users i guess and also the reason that we illustrate setting val to 0; this might be a good addition